signer/tests/integration/bitcoin_validation.rs

```diff
@@ -663,13 +738,27 @@ async fn outcome_is_independent_of_input_order() {
     let mut rng = OsRng;
     let (rpc, faucet) = regtest::initialize_blockchain();
 
-    let ctx = TestContext::builder()
+    let mut ctx = TestContext::builder()
         .with_storage(db.clone())
         .with_first_bitcoin_core_client()
         .with_mocked_stacks_client()
         .with_mocked_emily_client()
         .build();
 
+    ctx.with_emily_client(|client| {
```

Comment:
suggestion: just to make these cleaner (i.e. not take so much space) I might add a common function somewhere in `testing` that takes either a context or the stacks/emily clients and sets up this expect, like a `configure_zero_cap_limits()` or something, looks like we use exactly this block in a number of places both in this file and elsewhere.

Reply:
Now that the cap are cached, those are not needed anymore fortunately!